src/components/forms/select/RuiMenuSelect.vue

```diff
@@ -30,6 +31,7 @@ export interface Props {
   errorMessages?: string | string[];
   successMessages?: string | string[];
   showDetails?: boolean;
+  returnPrimitive?: boolean;
```

Comment:
Do we really need to expose yet another property?

Wouldn't it be better to return the `selectedItem`, or the `selectedItem[keyAttr]` is defined?

Reply:
so I did so because vuetify also has `return-object` prop, that basically says return an object when a value is selected... since we already return object by default, just thought to add `return-primitive` to have the same functionality... just in case we pass `key-attr` and we still need only the primitive value when a value is selected.